modeling.py

```diff
@@ -0,0 +1,146 @@
+import nimfa
+import pandas as pd
+from nfldb_tables import NFLFrames
+from scipy.sparse import csr_matrix as csrm
+
+
+class PositionNMF(object):
```

Comment:
Great.  Have fun climbing.  The computer I'm on right now doesn't have postgres installed, which is my current excuse for not writing code (couldn't test it).  Will look at this.  Have fun climbing.